ibtk/include/ibtk/FEProjector.h

```diff
@@ -45,10 +47,10 @@ class FEProjector
 {
 public:
     /// Constructor.
-    FEProjector(libMesh::EquationSystems* equation_systems, bool enable_logging = true);
+    FEProjector(libMesh::EquationSystems* equation_systems, bool enable_logging = true, int num_fischer_vectors = 5);
```

Comment:
Suggest keeping `enable_logging` as last argument --- if we are still doing this consistently.

Should `FEProjector` take in a `Database` for settings...?

Reply:
I think everything should take an input database - it makes dealing with these solver settings way easier.

Reply:
I added an input database. We don't enable logging when this object is set up in `FEDataManager` - I preserved that behavior.